source/f5_virtualserver.go

```diff
@@ -147,6 +148,12 @@ func (vs *f5VirtualServerSource) endpointsFromVirtualServers(virtualServers []*f
 	var endpoints []*endpoint.Endpoint
 
 	for _, virtualServer := range virtualServers {
+		if !isVirtualServerReady(virtualServer) {
+			log.Infof("F5 VirtualServer %s/%s is not ready or is missing an IP address, skipping endpoint creation.",
```

Comment:
```suggestion
			log.Warnf("F5 VirtualServer %s/%s is not ready or is missing an IP address, skipping endpoint creation.",
```
Shouldn't it be a warning ?

Reply:
Fixed here: https://github.com/kubernetes-sigs/external-dns/pull/4996/commits/d5c7f231712c895be2ad2ad284ddcb671ee22552